### PR TITLE
fix agent doc

### PR DESCRIPTION
**Pull request by arttor**

### CODE REVIEW

File: service/agent/README.md [hunk 1]
```diff
@@ -7,8 +7,11 @@ Can be used as alternative to [Chorus proxy](../proxy).
 
 ## Usage
 
-Set proper `url` and `fromStorage` params to [config](./config.yaml) and deploy agent to be reachable by source s3
-storage.
+Set proper `url`, `fromStorage`, and `redis` credentials to [config](./config.yaml) and deploy agent to be reachable by source s3
```
Comment (schucly) [inline service/agent/README.md:10]: Change `s3` to `S3`.

File: service/agent/README.md [hunk 2]
```diff
@@ -7,8 +7,11 @@ Can be used as alternative to [Chorus proxy](../proxy).
...
+storage. 
+
+> [!IMPORTANT]  
+> Make sure that `chorus-worker` and `chorus-agent` using the same Redis instance.
```
Comment (schucly) [inline service/agent/README.md:14]: Better:
Make sure that `chorus-worker` and `chorus-agent` are using the same Redis instance.

File: service/agent/README.md [hunk 3]
```diff
@@ -29,4 +32,4 @@ chorctl repl add --from="<src_storage_name>" --to="<dest_storage_name>" --user="
 
 - Amazon s3 bucket
   notifications [documentation](https://docs.aws.amazon.com/AmazonS3/latest/userguide/EventNotifications.html)
-- Ceph RGW s3 bucket notifications [documentation](https://docs.ceph.com/en/latest/radosgw/notifications/)
\ No newline at end of file
+- Ceph RGW s3 bucket notifications [documentation](https://docs.ceph.com/en/latest/radosgw/notifications/)
```
Comment (schucly) [inline service/agent/README.md:35]: Change `s3` to `S3`.

**Review by schucly — CHANGES_REQUESTED** [review]

File: service/agent/README.md (hunk 1 above)
Comment (arttor) [reply service/agent/README.md]: done

File: service/agent/README.md (hunk 3 above)
Comment (arttor) [reply service/agent/README.md]: done

File: service/agent/README.md (hunk 2 above)
Comment (arttor) [reply service/agent/README.md]: done

**Review by schucly — APPROVED** [review]

**Pull request merged**